salt/metalk8s/defaults.yaml

```diff
@@ -51,4 +53,36 @@ kubelet:
     authorization:
       mode: AlwaysAllow
 
+ca:
+  cert:
+    name: /etc/kubernetes/pki/ca.crt
+    signing_private_key: /etc/kubernetes/pki/ca.key
+    CN: kubenetes
+    keyusage: "critical digitalSignature, keyEncipherment, keyCertSign"
+    basicConstraints: "critical CA:true"
+    days_valid: 3650
+  signing_policies:
+    api:
```

Comment:
Let's give this a more 'unique' name.

Reply:
Like `kube-apiserver-policy` (don't know if dash allowed, but something like that ok)